packages/cloudflare/src/api/overrides/incremental-cache/kv-incremental-cache.ts

```diff
@@ -9,6 +11,17 @@ export const NAME = "cf-kv-incremental-cache";
 
 export const BINDING_NAME = "NEXT_INC_CACHE_KV";
 
+export type KeyOptions = {
+  isFetch: boolean;
+  buildId?: string;
```

Comment:
```suggestion
  buildId: string;
```

Reply:
which reminds me that I wanted to apply the default in the fn